planning/behavior_path_planner/autoware_behavior_path_lane_change_module/src/interface.cpp

```diff
@@ -452,5 +471,17 @@ void LaneChangeInterface::updateSteeringFactorPtr(
     {selected_path.info.shift_line.start, selected_path.info.shift_line.end},
     {output.start_distance_to_path_change, output.finish_distance_to_path_change},
     steering_factor_direction, SteeringFactor::APPROACHING, "");
+
+  const uint16_t planning_factor_direction = std::invoke([&output]() {
```

Comment:
```suggestion
  const auto planning_factor_direction = std::invoke([&output]() {
```
Just for code consistency purpose.